### DISCUSSION TITLE
Choose fields shown in Create Item window

### DISCUSSION THREAD

**Discussion by Openuser87** [sysadminsmedia/homebox#306] [Ideas]
### What is the problem you are trying to solve with this feature?

Say a user wants to enter name, description, purchase place and purchase price for every item. The latter two are not part of **Create Item** window. User has to click the **Create** button, then to to **Edit**, then scroll down to the price text fields. This is time consuming.

### What is the solution you are proposing?

In settings, let user choose (e.g. with checkboxes) the fields they find relevant for the **Create Item** window.

### What alternatives have you considered?

See no comfortable alternatives, as every user may need a different set of fields they want to "quick-enter" without going to the edit field

Maybe if [this](https://github.com/sysadminsmedia/homebox/issues/17) gets implemented, has an ability to add an item, **and** an ability to select columns, this would be also comfortable enough for quickly entering many items with relevant values. But it would be nice to have both ways.

### Contributions

- [x] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by tonyaellie** [comment]
![image](https://github.com/user-attachments/assets/5961a2a0-6408-4fde-aba6-3d526c845708)
![image](https://github.com/user-attachments/assets/ac05714e-5a8e-4b5e-b2a1-ea3934355458)
![image](https://github.com/user-attachments/assets/86b35661-8e6d-418a-b670-3ec574aecc35)

Maybe a UI like this?

**Reply by BoronFinder** [reply]
That looks good, but I would consider tieing this to the label instead of every item. E.g. for some kinds of items having a photo would be more relevant, for others a quantity and/or price, but no photo. (The same goes for custom labels, see here: https://github.com/sysadminsmedia/homebox/discussions/80 )
It would be necessary to enter the label beforehand, however.

**Comment by tonyaellie** [comment]
Closing this as part of our consolidation efforts to clean up feature requests, this will be included as part of https://github.com/sysadminsmedia/homebox/issues/435
